graph_nets/modules.py

```diff
@@ -134,7 +136,11 @@ def _build(self, graph):
       ValueError: If any of `graph.nodes`, `graph.edges`, `graph.receivers` or
         `graph.senders` is `None`.
     """
-    return self._node_block(self._edge_block(graph))
+    if edge_model_kwargs is None:
```

Comment:
(here and other changes in modules.py, except GraphIndependent) You don't need to replace the kwargs by empty dicts here, since it will be handled by the corresponding sub-blocks.